### ISSUE TITLE
IDISP004 should not warn when passing ctor argument that is disposed by the base class.

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#50]
```cs
namespace RoslynSandbox
{
    using Ninject;
    using Ninject.Modules;

    public static class Foo
    {
        public static IKernel Kernel { get; } = new FooKernel(
            new NinjectSettings(),
            new FooModule());
    }

    public class FooKernel : StandardKernel
    {
        public FooKernel(INinjectSettings settings, params INinjectModule[] modules)
            : base(settings, modules)
        {
        }
    }

    public class FooModule : NinjectModule
    {
        public override void Load()
        {
        }
    }
}
```

Not sure what is right here, maybe add `Ninject.StandardKernel` to spexial case list?
